### PR TITLE
Monacoin: Update hardcoded seeds

### PR DESCRIPTION
**Pull request by yamada-guro-baru**
I updated hardcoded seeds of core.
`seeds.tamami-foundation.org` with the help of @wakiyamap .

### CODE REVIEW

File: contrib/seeds/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ to addrman with).
 
 The seeds compiled into the release are created from poolers's DNS seed data, like this:
 
-    curl -s https://www.monacoinpool.org/seeds.txt > seeds_main.txt
+    curl -s http://seeds.tamami-foundation.org/seeds.txt > seeds_main.txt
```
Comment (wakiyamap) [inline contrib/seeds/README.md]: Use `https://seeds.tamami-foundation.org/seeds.txt`

**Review by wakiyamap — CHANGES_REQUESTED** [review]

File: contrib/seeds/README.md (hunk 1 above)
Comment (yamada-guro-baru) [reply contrib/seeds/README.md]: I fixed it.
I matched it to bitcoin.
https://github.com/bitcoin/bitcoin/blob/master/contrib/seeds/README.md

Comment (wakiyamap) [reply contrib/seeds/README.md]: great!

**Review by wakiyamap — APPROVED** [review]

**Comment by cryptcoin-junkey** [comment]
I've not tested but LGTM.

**Pull request merged**